source/fundamentals/crud/read-operations/limit.txt

```diff
@@ -109,14 +113,13 @@ Combining Skip and Limit
 ------------------------
 
 To see the next three longest books, append the ``skip()`` method to your
```

Comment:
[question]

Would it be helpful to specify that you are skipping three books here? "See the next three longest books" only makes sense in the context of the whole topic. If it said something like "to skip the first three books and see the next..." then it would make sense even if someone jumps here.

Reply:
Thanks for spotting that. I think this raises a point about what type of documentation this is (as well as other Fundamentals content) and whether we would expect people to start from a prior section such as in a tutorial or how-to. When this page was authored, I think the assumption (whether correct) was that the reader would be reading straight through and trying the code.

I'll create a ticket to try to capture a summary of your comment/request and link back to here so that we can address it later.

Feel free to create tickets when you noticed other similar items. For this PR, I want to limit the scope directly to the problem defined in the ticket since it was user-reported feedback on broken content.

Reply:
https://jira.mongodb.org/browse/DOCSP-29547